AUTHORS

```diff
@@ -169,18 +186,21 @@ Paweł Prażak                            pawelprazak@users.noreply.github.com
 Peiqi Shi                               uestc.shi@gmail.com
 Peter Slovak                            slovak.peto@gmail.com
 Philippe Lafoucrière                    philippe.lafoucriere@gmail.com
-Philipp Gniewosz                        philipp.gniewosz@posteo.de
+Philipp Gniewosz                        philipp.gniewosz@cegeka.de
 Pierre-Yves Aillet                      pyaillet@users.noreply.github.com
+Pranavi Roy                             pranvyr@gmail.com
 Qasim Sarfraz                           qasim.sarfraz@esailors.de
 Quentin Monnet                          quentin@isovalent.com
 Raghu Gyambavantha                      raghug@bld-ml-loan4.olympus.f5net.com
+Rahul Jadhav                            nyrahul@gmail.com
 Rajat Jindal                            rajatjindal83@gmail.com
 Ray Bejjani                             ray@isovalent.com
 Renat Tuktarov                          yandzeek@gmail.com
 Rene Zbinden                            rene.zbinden@postfinance.ch
 Robin Hahling                           robin.hahling@gw-computing.net
 Rodrigo Chacon                          rochacon@gmail.com
 Romain Lenglet                          rlenglet@google.com
+romanspb80                              romanspb@yahoo.com
```

Comment:
Another possible entry for `.mailmap` (65dd482bac770a310f374c4b33fbee12fc4ef08a):

```
Roman Ptitcyn <romanspb@yahoo.com>
```